README.md

```diff
@@ -1,5 +1,20 @@
 # Firefly
 
+## Intro
+
+Firefly is an open-source web-based UI library for astronomical data archive access and visualization developed in Infrared Processing and Analysis Center (IPAC) at [Caltech](https://caltech.edu).
```

Comment:
We are apparently (according to George) not supposed to spell out what "IPAC" stands for any longer.  I think @vandesai1 has some experience crafting this introductory sentence without doing so; I always find it difficult.